main.go

```diff
@@ -69,7 +79,11 @@ func chanHandler(feed *rss.Feed, newchannels []*rss.Channel) {
 func makeHandler(master *feeds.Feed, sourceName string) rss.ItemHandlerFunc {
 	return func(feed *rss.Feed, ch *rss.Channel, items []*rss.Item) {
 		for i := 0; i < len(items); i++ {
-			published, _ := items[i].ParsedPubDate()
+			published, err := items[i].ParsedPubDate()
+			if err != nil {
+				log.Fatal(err)
```

Comment:
After more thought, I like the idea of skipping it. I see what you're saying, skip the whole item to remove it from the feed. Keeps invalid data out of the feed while still returning a valid feed.